src/transformers/__init__.py

```diff
@@ -1084,6 +1084,7 @@
             "AutoModelForSequenceClassification",
             "AutoModelForSpeechSeq2Seq",
             "AutoModelForTableQuestionAnswering",
+            "AutoModelForTextEncoding",
```

Comment:
The TF equivalent class should be in this init too

Reply:
Resolved in https://github.com/huggingface/transformers/pull/24305/commits/8b003f7d01556b0424550c784149f7d775989e3e